### DISCUSSION TITLE
v0.8.2 - Ruleset Manager

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#101] [Announcements]
## Breaking Changes (sorry Semantic Versioning)
- Move to `.NET 8` Desktop Runtime, instead of .NET 7
- Remove `Id` property in Rulesets (No impact)

## Application Updates
- Introducing `Ruleset Manager` Beta 🎉 - New feature for editing Rulesets/Rules, You can open it from Browser Selection window. (In future, It will be available as a desktop shortcut and Rest of the settings like Editing Browsers, Visual etc will be available from this one unified window for editing)
- Add new Rule Type `Domain`
- Add few Tooltips in Browser Selection Window (#89)
- Fix Hurl running in the background even after Crashing (#97)

## Development Updates
- Update `WPF-UI` to `v3.0.1`

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.8.1...v0.8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.8.2'>v0.8.2 - Ruleset Manager</a>.</em>

**Comment by Jay-o-Way** [comment]
> sorry Semantic Versioning

Haha, yah, this should be 0.9 🤓
